dev/com.ibm.ws.jaxrs.2.0.client/bnd.bnd

```diff
@@ -48,11 +48,11 @@ Service-Component: \
 instrument.classesExcludes: com/ibm/ws/jaxrs20/client/internal/resources/*.class
     
 -buildpath: \
+	com.ibm.ws.jaxrs.2.0.common;version=latest,\
```

Comment:
This is required so that when we depend on Liberty-specific overrides of CXF code, we load the overridden classes ahead of the CXF classes on the compiler's classpath.